python/versioneer.py

```diff
@@ -176,7 +176,7 @@
 
 * Source trees which contain multiple subprojects, such as
   [Buildbot](https://github.com/buildbot/buildbot), which contains both
-  "master" and "slave" subprojects, each with their own `setup.py`,
+  "main" and "slave" subprojects, each with their own `setup.py`,
```

Comment:
Since this file is automatically generated by the  `versioneer` tool, I've removed it from our changes. Our changes would likely be overwritten if we ever updated our `versioneer` version so I figured it's best to let them handle the contents of this file than to worry about these semantics ourselves.